<launch> 
  <node pkg="erp42_driver" name="erp42_serial" type="erp42_serial.py" output="screen">
    <param name="port" value="/dev/ERP-serial" />
  </node>

  <node pkg="erp42_driver" name="erp42_driver" type="erp42_driver.py" output="screen">
    <param name="p_gain" value="3" />
    <param name="i_gain" value="1" />
    <param name="d_gain" value="1" />
    <param name="RightSteerErrCal" value="0.8254" />
    <param name="LeftSteerErrCal" value="0.7202" />
    <param name="SteerOffset" value="0" />
    <param name="BrakeThresholdErr" value="-1" />
    <param name="BrakeLimit" value="100" />
  </node>
</launch>
  
